### PR TITLE
testing/grepcidr: new aport

### PR DESCRIPTION
**Pull request by vincentmele**
http://www.pc-tools.net/unix/grepcidr/
Filter IPv4 and IPv6 addresses matching CIDR patterns

### CODE REVIEW

**Comment by fcolista** [comment]
Thanks for your contribution!
Before applying this patch, I've some comments:

`arch="x86_64"`
There's a strong reason why only x86_64 is enabled?
You can set "all" in order to build this package for all the architecture supported by Alpine.
See: [APKBUILD:arch](https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#arch)

`options="!check"`
There's a reason why check is disabled? 

`source="http://www.pc-tools.net/files/unix/grepcidr-2.0.tar.gz"`
We'd prefer to use variables here, so when will come to bump a new version, it's enough changing the $pkgver. So, the filename should be $pkgname-$pkgver.tar.gz
See:[APKBUILD:source](https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#source)

`mv "$builddir"/grepcidr "$pkgdir"/bin`
This should go to /usr/bin rather than /bin.

Thanks a lot!
.:Francesco

**Comment by vincentmele** [comment]
Thank you for very much for reviewing and suggesting changes! I will make the changes and update this request.

File: testing/grepcidr/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+check() {
+	cd "$builddir"
+	if [ -x grepcidr ]; then echo OK; else exit 1; fi
```
Comment (jirutka) [inline testing/grepcidr/APKBUILD]: This is not an useful check. Could you run grepcidr, e.g. `./grepcidr --version`?

Comment (kaniini) [reply testing/grepcidr/APKBUILD]: Better yet, it would be good to run grepcidr against some sample data and confirm it behaves as expected.

**Pull request closed without merging**